packages/api-client/src/apiService.ts

```diff
@@ -33,6 +33,25 @@ export interface ShopwareApiInstance {
   _axiosInstance: AxiosInstance;
 }
 
+/**
+ * Check if api-client has minimum settings set
+ *
+ * @throws {Error}
+ *
+ * @param initialConfig {ClientSettings}
+ */
+function checkMinimumConfig(initialConfig: ClientSettings): void {
+  const getMessage = (missing: string) =>
+    `${missing} was not provided. Check your api-client settings. Read more at https://github.com/shopware/frontends/issues/276`;
```

Comment:
It would be better to link to docs with proper config setup information instead of the issue, which only tells to remove default config

Reply:
currently it tells not much indeed, but I will elaborate about it more there in the issue for some reasons:
- there is no dedicated page for api-client in our docs (except reference) as it's becoming replaced by api-client-next
- not much in readme there as well

or I can put some info in the home page of a package reference in docs 🤔